src/components/home/InstagramNewsSection.tsx

```diff
@@ -1,9 +1,28 @@
+import type { IGFeedType } from '../../AppTypes';
 import { SectionContainer } from '../shared/SectionContainer';
 
-export const InstagramNewsSection = () => {
+type Props = {
+  igFeed: IGFeedType[];
+};
+
+export const InstagramNewsSection = ({ igFeed }: Props) => {
   return (
     <SectionContainer id="novinky">
       <h2 className="text-blue text-h2-reversed">Novinky</h2>
+      <div className="flex flex-wrap gap-6 py-8 justify-center">
+        {igFeed.slice(0, 8).map(({ id, media_url, permalink }) => (
+          <a className="size-[200px] lg:size-[250px]" key={id} href={permalink} target="_blank">
```

Comment:
Hodil by se nějaký hover effect :P, nějaký `group-hover:scale-105` na img element,  a `group overflow-hidden` na link element a hotovson

Reply:
jop, vykonam. Token budeme musiet refreshnut no. Len neviem, ci nam to pojde spravit skrz CICD kedze je ten token ulozeny v github secrets